# aggregated info of an object
extended_object_detection/BaseObject complex_object

# sub parts of complex objects
extended_object_detection/BaseObject[] simple_objects
